.github/workflows/build.yml

```diff
@@ -5,6 +5,17 @@ on:
     branches: [main]
   pull_request:
     branches: [main]
+    paths:
```

Comment:
Are we sure these are exhaustive?

Reply:
not 100% sure, but I think it should cover everything I can think of now.